dali/base/dadfs.cpp

```diff
@@ -7363,11 +7414,23 @@ IDistributedFile *CDistributedFileDirectory::dolookup(CDfsLogicalFileName &_logi
                 CFileLock fcl;
                 unsigned mode = RTM_LOCK_READ | RTM_SUB;
                 if (hold) mode |= RTM_LOCK_HOLD;
+                // initWithFileLock() below will loop (releasing and relocking fcl) if it cannot get SuperOwner lock
+                // MCK - another way to solve this could be to use RTM_LOCK_WRITE for fcl
```

Comment:
Well, not really as discussed offline. It would then block whilst others are reading etc.
Should remove this comment as misleading.